sorc/orog_mask_tools.fd/inland.fd/inland.F90

```diff
@@ -78,14 +78,19 @@ PROGRAM inland_mask
 !! @author Ning Wang  
 SUBROUTINE mark_global_inland(cs_res)
   INTEGER, INTENT(IN) :: cs_res
+  INTEGER :: i_seed, j_seed
  
   ALLOCATE(done(cs_res,cs_res,6))
   ALLOCATE(inland(cs_res,cs_res,6))
   done = .false.
   inland = 1.0
-  i_ctr = cs_res/2; j_ctr = cs_res/2
 
-  CALL mark_global_inland_rec_d(i_ctr, j_ctr, 2, 0)
+  i_seed = cs_res/2; j_seed = cs_res/2
+  CALL mark_global_inland_rec_d(i_seed, j_seed, 2, 0)
+
+! to make sure black sea is excluded
```

Comment:
A bit more elaboration on how please

Reply:
I think doxygen should have this ?. Is there a reason i_ctr has been changed to i_seed ?

Reply:
The location of i_seed only matters for some experimental regional domains.

Reply:
No particular reason to change "i_ctr" to "i_seed" - just a better variable name.